awx/ui_next/src/screens/InstanceGroup/InstanceGroupList/InstanceGroupListItem.jsx

```diff
@@ -108,6 +109,13 @@ function InstanceGroupListItem({
                   <b>{instanceGroup.name}</b>
                 </Link>
               </span>
+              {instanceGroup.is_isolated ? (
+                <span css="margin-left: 12px">
+                  <Chip key={instanceGroup.id} isReadOnly>
+                    {i18n._(t`Isolated`)}
+                  </Chip>
```

Comment:
```suggestion
                  <Label key={instanceGroup.id} >
                    {i18n._(t`Isolated`)}
                  </Label>
```

Reply:
I also need to update this in the detail page. Thanks for review @jakemcdermott .